### PR TITLE
Adding base-url support

### PR DESCRIPTION
**Pull request by mdeggies**
This commit adds export support for all Stormpath environments. It also includes a small README update. :)

### CODE REVIEW

**Comment by mdeggies** [comment]
@rdegges no major rush, but we do have an enterprise customer waiting on this one ^^

File: stormpath-export [hunk 1]
```diff
@@ -8,7 +8,6 @@ Easily export your Stormpath (https://stormpath.com/) user data.
 Usage:
   stormpath-export configure
   stormpath-export [(<location> | -l <location> | --location <location>)]
-  stormpath-export [(-b <base_url> | --base-url <base_url>)]
```
Comment (rdegges) [inline stormpath-export:11]: Heyo! We need to leave this in so someone can supply the base url via the command line if they need to.

File: stormpath-export [hunk 2]
```diff
@@ -40,16 +39,16 @@ class StormpathExport(object):
 
     EXPORTS = ['tenants', 'applications', 'directories', 'groups', 'organizations', 'accounts']
 
-    def __init__(self, base_url=None):
```
Comment (rdegges) [inline stormpath-export:43]: I think we also need to leave this in -- this way if someone specifies a base url via the command line, we can pass it into this class and use it during the setup of the client.

File: stormpath-export [hunk 3]
```diff
@@ -40,16 +39,16 @@ class StormpathExport(object):
...
+    def __init__(self):
         """Initialize our Stormpath client, or die tryin' >:)"""
         if exists(CONFIG_FILE):
             credentials = loads(open(CONFIG_FILE, 'rb').read())
             self.client = Client(api_key={
                 'id': credentials.get('stormpath_api_key_id'),
                 'secret': credentials.get('stormpath_api_key_secret'),
-            }, base_url=base_url or None)
+            }, base_url=credentials.get('stormpath_base_url') or None)
```
Comment (rdegges) [inline stormpath-export:49]: I think this should be `credentials.get(...) or base_url or None` in that order. So we try to get it from the credentials file, or the CLI options

**Comment by rdegges** [comment]
Heyo, because of time constraints -- I'm going to merge this in now and make some updates myself. Hope you don't mind! We can review this whenever.

**Pull request merged**